packages/browser/src/core/user/index.ts

```diff
@@ -62,15 +62,14 @@ const ONE_YEAR = 365
 
 export class Cookie extends Store {
   static available(): boolean {
-    let cookieEnabled = window.navigator.cookieEnabled
-
-    if (!cookieEnabled) {
-      jar.set('ajs:cookies', 'test')
-      cookieEnabled = document.cookie.includes('ajs:cookies')
-      jar.remove('ajs:cookies')
+    try {
+      jar.set('ajs_cookies', 'test')
```

Comment:
nit: for quick reuse putting `ajs_cookies` as a const `AJS_COOKIE_AVAILABILITY_KEY` or similar.

I would also suggest renaming it to `ajs_cookies_check` or something to highight that it is only an availability check cookie.

Reply:
good point! All of these strings get repeated in the bundle -- using a constant saves bytes!

Reply:
Updated